frontend/src/components/smart/PvPArena.vue

```diff
@@ -212,8 +228,41 @@ export default {
       };
     },
 
-    kickCharacterFromArena() {
+    async kickCharacterFromArena() {
       this.isCharacterInArena = false;
+
+      const kickedEvents = await this.getKickedEvents(this.contracts());
+
+      if (kickedEvents.length) {
+        const formattedResult = formatCharacterKickedEvent(kickedEvents[kickedEvents.length - 1].returnValues);
+
+        if (+localStorage.getItem(`${getCharacterNameFromSeed(this.currentCharacterId)}: lastKickedTime`) < +formattedResult.timestamp) {
```

Comment:
Why not just use the char ID for the key? (And store the key in a variable if you are gonna reuse it)